core/app/repositories/uffizzi_core/deployment_repo.rb

```diff
@@ -8,7 +8,7 @@ module UffizziCore::DeploymentRepo
       where(name: name)
     }
     scope :with_amazon_repos, -> { includes(containers: [:repo]).where(containers: { repos: { type: UffizziCore::Repo::Amazon.name } }) }
-    scope :existed, -> { where(state: [:active, :failed]) }
+    scope :actual, -> { where(state: [:active, :failed]) }
```

Comment:
Why have we changed from `existed` to `actual`?

Reply:
It is more descriptive than the past form of "exist".
Why not existing: `disabled` are existing too.